api/include/opentelemetry/trace/span_context_kv_iterable.h

```diff
@@ -30,5 +30,22 @@ class SpanContextKeyValueIterable
    */
   virtual size_t size() const noexcept = 0;
 };
+
+/**
+ * @brief Null Span context that does not carry any information.
+ */
+class NullSpanContext : public SpanContextKeyValueIterable
+{
+public:
+  bool ForEachKeyValue(
+      nostd::function_ref<bool(SpanContext, const opentelemetry::common::KeyValueIterable &)>
+          callback) const noexcept override
+  {
+    return false;
```

Comment:
`false` signals an error during iterating over elements. `true` should be returned instead.

Reply:
Got it. I'll fix it.